topology/Rutgers University/Rutgers OARC/SITE.yaml

```
@@ -0,0 +1,8 @@
+AddressLine1: 136 Frelinghuysen Road
+City: Piscataway
+Country: United States
+ID: 10307
+Latitude: 40.522918
+Longitude: -74.464514
+State: New Jersey
+Zipcode: 08854-8019
```

```suggestion
Zipcode: '08854-8019'
```
probably not necessary in this one case, but it's a good habit to keep up.